docs/theming.md

```diff
@@ -430,37 +430,35 @@ Since our cards only contain text and no components, let's keep it simple for no
 > Note: in the future we plan to provide a Javascript utility method for changing all derived colors and making this
 use-case easier.
 
+## Custom Sass Mixins
 
-## Dark Themes
+We also have provided a way to customize on a per component basis. We realize that you may want a button with a
+blue background and another with a red background on the same page. Each component's package is documented with all
+our provided sass mixins and what arguments to pass them. For example, here is a link to our [Button Sass Mixins docs](https://github.com/material-components/material-components-web/tree/master/packages/mdc-button#advanced-sass-mixins).
```

Comment:
* "all our provided" -> "all available"
* "what arguments to pass them" -> "what parameters they accept"
* Use a relative link to the button docs (../packages/mdc-button#advanced-sass-mixins should work) and rephrase to "See the MDC Button Sass mixins for example." (with link around "MDC Button Sass mixins")